### ISSUE TITLE
[MHV Medications / Accessibility] Accordion button and filter buttons do not respond to text spacing

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#125176]
### 📝 User story

As a user, I want text to use my text spacing settings, so that I can adapt my display as needed to read and understand text content.

### 📝 Problem statement

Labels for the Filter accordion control and the Apply and Clear buttons does not apply text spacing settings.

### 🌌 Supporting screenshots or videos

<img width="630" height="797" alt="Screenshot of Filter accordion showing some text with spacing and controls without spacing" src="https://github.com/user-attachments/assets/23896482-00c4-475c-9cc7-13b899a2e68e" />

The Medications Filter feature with the Filter, Apply, and Clear controls without text spacing. Note that the prefilled text in the Select menu also does not adapt to text spacing settings.

### 👣 Reproduction steps

1. Navigate to the [Medications list page](https://staging.va.gov/my-health/medications?page=1) (username: (mhvmark.test@id.me).
2. Use the ARC Toolkit browser extension to check text spacing.
3. Notice that some content applies the text spacing and other content does not.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards

- [Guideline 1.4 Distinguishable: Make it easier for users to see and hear content including separating foreground from background.](https://www.w3.org/WAI/WCAG22/Understanding/distinguishable)
 - [SC 1.4.12 Text Spacing (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing)

#### Experience Standards
- [Comprehension: User easily understands the product](https://design.va.gov/about/experience-standards/#category-comprehension)
  - [User can perceive all meaningful elements](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)

### 𝌣 Proposed solution or next steps

Explore whether there are CSS and browser defaults that are keeping changes to text spacing from applying to buttons and inputs.

### ✅ Acceptance criteria

- [ ] The system adapts all displayed text to custom text spacing settings.
- [ ] The user changes text spacing settings and sees the change applied to all elements that include text.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by sarahhortonadhoc** [comment]
I tested using @jasonday’s VA11y bookmarklet and it seems that these components do respond to text spacing. If you have a minute, please confirm and I'll close the ticket — thank you!

**Comment by sarahhortonadhoc** [comment]
The issue here is text spacing: [SC 1.4.12 Text Spacing (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing). I did initial testing with ARC Toolkit, but when I tested using VA11y all the text responded. Is text spacing also something you're tracking as part of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4754? It seems like it might not be an issue.
